src/util/removeEmpty.spec.js

```diff
@@ -0,0 +1,25 @@
+import assert from 'assert';
+import removeEmpty from './removeEmpty';
+
+describe('removeEmpty', () => {
+    context('with no empty values', () => {
```

Comment:
we don't use `context()` (it's just an alias for `describe()`. Please replace with `describe()`.